e2e/rbd.go

```diff
@@ -232,6 +232,15 @@ func checkClusternameInMetadata(f *framework.Framework, ns, pool, image string)
 	}
 }
 
+// ByFileAndBlockEncryption wraps ginkgo's By to run the test body using file and block encryption specific validators.
+func ByFileAndBlockEncryption(
```

Comment:
this is much appreciated, less copy/pasting all around! We should do something like this more often to cleanup the e2e code.